target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -163,6 +163,11 @@ case "$FIRMWARE" in
 		ath10kcal_extract "art" 20480 2116
 		ath10kcal_patch_mac $(macaddr_add $(mtd_get_mac_binary info 8) +1)
 		;;
+	tplink,re355-v1|\
```

Comment:
This ordering has to be moved one block up (`tplink,re450-v2` > `tplink,re355-v1`)

Reply:
Fixed.